apps/tangle-dapp/containers/NominatorStatsContainer/NominatorStatsContainer.tsx

```diff
@@ -64,6 +74,32 @@ export const NominatorStatsContainer: FC = () => {
     }
   }, [substrateAddress]);
 
+  const unbondingRemainingErasTooltip = useMemo(() => {
+    if (unbondingRemainingErasError) {
+      notificationApi({
+        variant: 'error',
+        message: unbondingRemainingErasError.message,
+      });
+    }
+
+    if (!unbondingRemainingErasData?.value1) return null;
+
+    if (unbondingRemainingErasData.value1.length === 0) {
+      return 'You have no unbonding tokens.';
+    }
+
+    const elements = unbondingRemainingErasData.value1.map((era, index) => (
```

Comment:
This can be changed to:

```
const elements = unbondingRemainingErasData.value1.map((era, index) => (
  <div key={index} className="text-center mb-2">
    <p>Unbonding {era.amount}</p>
    <p>{era.remainingEras} eras remaining</p>
  </div>
));

return {elements};
```